contrib/fiber/fiber.go

```diff
@@ -0,0 +1,70 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
Based on the path patterns that we [conventionally use](https://github.com/DataDog/dd-trace-go/blob/v1/contrib/README.md) the correct folder should be `gofiber/fiber.v2`

Reply:
done